src/js/_enqueues/vendor/jquery/jquery-migrate.min.js

```diff
@@ -1,2 +1,3 @@
+//# sourceMappingURL=jquery-migrate.min.map
```

Comment:
I'm going to go ahead and commit this so we don't get stuck, but just as a code review note, I'm going to remove the map reference :) Almost missed it because GH suppresses diffs on minified files by default...

Reply:
Makes sense. Thanks!